packages/@atjson/renderer-commonmark/src/index.ts

```diff
@@ -689,7 +690,10 @@ export default class CommonmarkRenderer extends Renderer {
   /**
    * Paragraphs are delimited by two or more newlines in markdown.
    */
-  *Paragraph(): Generator<void, string, string[]> {
+  *Paragraph(
+    _paragraph: Paragraph,
```

Comment:
what's up with these arguments?

Reply:
Our upstream repos will whine and say it's not a proper subclass.